sigstore-testkit/src/main/java/dev/sigstore/testkit/tuf/TestResources.java

```diff
@@ -25,8 +25,12 @@
 
 public class TestResources {
 
-  public static final Path UPDATER_TRUSTED_ROOT = Path.of(Resources.getResource("dev/sigstore/tuf/trusted-root.json").getPath());
-  public static final Path TUF_TEST_DATA_DIRECTORY = UPDATER_TRUSTED_ROOT.getParent();
+  public static final Path UPDATER_REAL_TRUSTED_ROOT = Path.of(Resources.getResource("dev/sigstore/tuf/real/trusted-root.json").getPath());
+  public static final Path UPDATER_SYNTHETIC_TRUSTED_ROOT = Path.of(Resources.getResource("dev/sigstore/tuf/synthetic/trusted-root.json").getPath());
+
+  // get the child folder and retrieve the parent to avoid resource loader finding the dev/sigstore/tuf folder in the classes/ dir.
```

Comment:
perhaps we need some better organization.

Reply:
for another time though...

Reply:
can you just add a todo with an issue number linking to this in the issues?

Reply:
I've added a todo. I actually know how I can clean this up once I update the root in main.